push_notifications/migrations/0008_fcmdevice.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.1.12 on 2021-07-08 05:32
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('push_notifications', '0007_uniquesetting'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='FCMDevice',
+            fields=[
+                ('gcmdevice_ptr', models.OneToOneField(auto_created=True, on_delete=django.db.models.deletion.CASCADE, parent_link=True, primary_key=True, serialize=False, to='push_notifications.gcmdevice')),
+                ('platform', models.CharField(blank=True, choices=[('i', 'ios'), ('a', 'android'), ('w', 'web')], help_text='Optional device platform: i - ios, a - android, w - web', max_length=1, null=True)),
```

Comment:
I'd much rather see a small positive integer field for faster filtering on the database.

Reply:
That sounds like a good idea indeed.